### ISSUE TITLE
Can't set controller_name/action on service

### ISSUE THREAD

**Issue by ryfow** [mattetti/wd-sinatra#11]
WD supports setting controller_name and action attributes on the service for routing to implementation code. wd_sinatra supports setting a block as the implementation with the `implementation(&block)` method. As far as I can tell, wd_sinatra doesn't pay attention to the `controller_name` and `action` attributes.

I don't know if one way or the other is better, but it's confusing right now. Supporting something like `service.implementation ModuleName, :method_name` across the board might be nice.

**Comment by mattetti** [comment]
That is correct, wd_sinatra is currently opinionated and only supports the block approach. This implementation https://github.com/mattetti/Weasel-Diesel/blob/master/lib/framework_ext/sinatra.rb  however does support the controller approach (which I personally don't prefer). 

I don't think it's a good idea to mix both approach in the same app, so what we could do, is to make it a generator option if you are interested in taking care of that.

**Comment by ryfow** [comment]
I see. I think the confusing part for me is that `controller_name` and `action` are right in [weasel_diesel.rb](https://github.com/mattetti/Weasel-Diesel/blob/master/lib/weasel_diesel.rb#L91) as well being mentioned in [README.md](https://github.com/mattetti/Weasel-Diesel/blob/master/README.md) making them seem like a core part of the DSL. Perhaps they should be stuck in [framework_ext](https://github.com/mattetti/Weasel-Diesel/tree/master/lib/framework_ext).

**Comment by mattetti** [comment]
Very good point, would you mind sending a pull request?
